cpp/tests/CMakeLists.txt

```diff
@@ -227,6 +227,15 @@ set(FA2_TEST_SRC
 
 ConfigureTest(FA2_TEST "${FA2_TEST_SRC}" "")
 
+###################################################################################################
+# - TSP  tests --------------------------------------------------------------------------
+
+set(TSP_TEST_SRC
+    "${CMAKE_SOURCE_DIR}/../thirdparty/mmio/mmio.c"
```

Comment:
We're changing lines like these to use `CMAKE_CURRENT_SOURCE_DIR` instead (which means the relative path will be `"${CMAKE_CURRENT_SOURCE_DIR}/../../thirdparty/mmio/mmio.c"` to support using cugraph as a 3rd party package, as well as to be consistent with other RAPIDS repos.

Reply:
Changed it here: https://github.com/rapidsai/cugraph/pull/1360/commits/e099b5b9479ed33525710bd27f9952f7c4b1febd